fe/src/pages/user/login/Login.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  const navigate = useNavigate();
+
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    axios
+      .post('http://127.0.0.1:8000/api/login', {
+        ...formData,
+      })
+
+      .then(() => {
+        navigate('/');
+      })
+
+      .catch((error) => {
+        setErrorMessage(error.response.data.message);
+      });
```

Comment:
- do not put newline on this.
```suggestion
    axios
      .post('http://127.0.0.1:8000/api/login', {
        ...formData,
      })
      .then(() => {
        navigate('/');
      })
      .catch((error) => {
        setErrorMessage(error.response.data.message);
      });
```